l10n_es_ticketbai_api/models/ticketbai_invoice_line.py

```diff
@@ -40,32 +40,35 @@ def _check_description(self):
     def _check_quantity(self):
         for record in self:
             tbai_utils.check_str_decimal(_(
-                "TicketBAI Invoice %s: Line %s Quantity"
-            ) % (record.tbai_invoice_id.name, record.description), record.quantity)
+                "TicketBAI Invoice %s: Line %s Quantity")
```

Comment:
Por qué hacer estos cambios (hay bastantes) que ensucian el diff y no aportan nada?

Reply:
Arreglado.

Una pregunta, ¿por que en nuestro repo aparece que he hecho yo el push, pero después aquí indica que ha sido un compañero mío? Estamos mirando por todas partes pero no sacamos la explicación.

Reply:
Yo en el commit veo que pone cambios de ao-landoo (Aritz Olea). ¿No eres tú? Sí que el committer pone `ig-landoo`, pero eso será porque desde donde hacéis push tenéis puesto el correo de esa otra persona en git. Revisad configuración de vuestro git, que hay 2 configuraciones de identidad.